src/linter/tests/rules/invalid-change-version.test.mjs

```diff
@@ -5,16 +5,21 @@ import { assertEntry } from '../fixtures/entries.mjs';
 
 describe('invalidChangeVersion', () => {
   it('should return an empty array if all change versions are valid', () => {
-    const issues = invalidChangeVersion(assertEntry);
+    const issues = invalidChangeVersion([assertEntry]);
 
     deepEqual(issues, []);
   });
 
   it('should return an issue if a change version is invalid', () => {
-    const issues = invalidChangeVersion({
-      ...assertEntry,
-      changes: [...assertEntry.changes, { version: ['v13.9.0', 'REPLACEME'] }],
-    });
+    const issues = invalidChangeVersion([
+      {
+        ...assertEntry,
+        changes: [
+          ...assertEntry.changes,
+          { version: ['v13.9.0', 'REPLACEME'] },
```

Comment:
Not related to this PR, but FWIW it's unrealistic to have a specific version AND `REPLACEME`, either a feature has been released (and `REPLACEME` has been replaced), either it has not been released yet

Reply:
That's for testing reasons, and even so, it's being changed in #238